src/dowel/tensor_board_output.py

```diff
@@ -106,3 +132,20 @@ def dump(self, step=None):
     def close(self):
         """Flush all the events to disk and close the file."""
         self._writer.close()
+
+
+class NonexistentAxesError(LoggerWarning):
```

Comment:
if these are going to be warnings, they should not be raised as exceptions. You can use `warnings.warn` to print these without interrupting flow control.